src/DataFormatter/FormEncodedDataFormatter.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace SilverStripe\RestfulServer\DataFormatter;
+
+/**
+ * Accepts form encoded strings and converts them
+ * to a valid PHP array via {@link parse_str()}.
+ *
+ * Example when using cURL on commandline:
+ * <code>
+ * curl -d "Name=This is a new record" http://host/api/v1/(DataObject)
+ * curl -X PUT -d "Name=This is an updated record" http://host/api/v1/(DataObject)/1
+ * </code>
+ *
+ * @todo Format response form encoded as well - currently uses XMLDataFormatter
+ *
+ * @author Cam Spiers <camspiers at gmail dot com>
+ *
+ * @package framework
+ * @subpackage formatters
```

Comment:
Later: Let's remove the `@package` and `@subpackage` annotations from these files, since they're no longer part of framework and namespaces replace the need for the subpackage